packages/core/android/src/main/java/com/segment/analytics/reactnative/core/RNAnalyticsModule.kt

```diff
@@ -196,14 +207,6 @@ class RNAnalyticsModule(context: ReactApplicationContext): ReactContextBaseJavaM
             return promise.reject("E_SEGMENT_ERROR", e)
         }
 
-        if(options.getBoolean("trackAppLifecycleEvents")) {
-            this.trackApplicationLifecycleEvents(writeKey)
```

Comment:
Could you explain this change?
After removing this line the private method defined in line 65 isn't called anywhere, sounds like a bug?

Reply:
yup this was a bug. @nfcampos thanks for pointing it out :), It really helped us solve a crucial bug 🙌